src/main/resources/static/js/app.js

```diff
@@ -31,6 +31,7 @@ $(document).ready(
 
                     },
                     error: function () {
+                        // TODO: think about displaying the error from server
```

Comment:
There's a TODO here

Reply:
It's not really a TODO to fix now but rather a thing that we might want to do in the future.

To put everything into context, that code was already like that from the beginning. What that code does is that if some error is returned by the server (it does not matter which type of error: 4xx, 5xx, ...) it shows in red letters "SOMETHING WENT WRONG". The purpose of that TODO is to remember that we might want to show those errors to inform the user what could be the cause of the error instead of just saying an error has occured, but right now we have better things to do than spent time prettifying things that do not matter (front-end).